csunplugged/config/settings/base.py

```diff
@@ -158,15 +158,16 @@
 # STATIC FILE CONFIGURATION
 # ------------------------------------------------------------------------------
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#static-root
-STATIC_ROOT = str(ROOT_DIR.path("staticfiles"))
+STATIC_ROOT = str(ROOT_DIR.path("staticfiles")) + "/"
```

Comment:
Also, probably better to do `os.path.join(ROOT_DIR.path("staticfiles"), "")` - that will add slash only if it is missing.